app/models/spree/stock_location_decorator.rb

```diff
@@ -1,9 +1,19 @@
 Spree::StockLocation.class_eval do
+  validate :just_default_location
```

Comment:
i didnt get the "just" quickly. maybe ensure_single_stock_location is easier to understand?